src/analysis/complex/isometry.lean

```diff
@@ -145,3 +147,24 @@ begin
   { simpa using eq_mul_of_inv_mul_eq h₁ },
   { exact eq_mul_of_inv_mul_eq h₂ }
 end
+
+/-- The matrix representation of `rotation`. -/
```

Comment:
```suggestion
/-- The matrix representation of `rotation a` is equal to the conformal matrix `![![a, -b]. ![b, a]]`. -/
```

Reply:
I've made a different doc string change there to state the matrix more precisely.

Reply:
Oops! Thanks for catching my wrong suggestion. That was a bit too hasty.